[2022-03-15 11:48:21,343] {taskinstance.py:1037} INFO - Dependencies all met for <TaskInstance: simple_dag.test scheduled__2022-03-15T11:45:00+00:00 [queued]>
[2022-03-15 11:48:21,350] {taskinstance.py:1037} INFO - Dependencies all met for <TaskInstance: simple_dag.test scheduled__2022-03-15T11:45:00+00:00 [queued]>
[2022-03-15 11:48:21,350] {taskinstance.py:1243} INFO - 
--------------------------------------------------------------------------------
[2022-03-15 11:48:21,350] {taskinstance.py:1244} INFO - Starting attempt 1 of 1
[2022-03-15 11:48:21,350] {taskinstance.py:1245} INFO - 
--------------------------------------------------------------------------------
[2022-03-15 11:48:21,364] {taskinstance.py:1264} INFO - Executing <Task(BashOperator): test> on 2022-03-15 11:45:00+00:00
[2022-03-15 11:48:21,369] {standard_task_runner.py:52} INFO - Started process 21469 to run task
[2022-03-15 11:48:21,376] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'simple_dag', 'test', 'scheduled__2022-03-15T11:45:00+00:00', '--job-id', '292', '--raw', '--subdir', 'DAGS_FOLDER/simple_dag.py', '--cfg-path', '/var/folders/tl/ljh9dpdd2fnbgmc62qc2b49h0000gn/T/tmp40ro0yvb', '--error-file', '/var/folders/tl/ljh9dpdd2fnbgmc62qc2b49h0000gn/T/tmpjh91rv31']
[2022-03-15 11:48:21,379] {standard_task_runner.py:77} INFO - Job 292: Subtask test
[2022-03-15 11:48:21,448] {logging_mixin.py:109} INFO - Running <TaskInstance: simple_dag.test scheduled__2022-03-15T11:45:00+00:00 [running]> on host 1.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.ip6.arpa
[2022-03-15 11:48:21,497] {taskinstance.py:1429} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=simple_dag
AIRFLOW_CTX_TASK_ID=test
AIRFLOW_CTX_EXECUTION_DATE=2022-03-15T11:45:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-03-15T11:45:00+00:00
[2022-03-15 11:48:21,499] {subprocess.py:62} INFO - Tmp dir root location: 
 /var/folders/tl/ljh9dpdd2fnbgmc62qc2b49h0000gn/T
[2022-03-15 11:48:21,500] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'echo "The DAG has ran!"']
[2022-03-15 11:48:21,510] {subprocess.py:85} INFO - Output:
[2022-03-15 11:48:21,515] {subprocess.py:89} INFO - The DAG has ran!
[2022-03-15 11:48:21,516] {subprocess.py:93} INFO - Command exited with return code 0
[2022-03-15 11:48:21,542] {taskinstance.py:1272} INFO - Marking task as SUCCESS. dag_id=simple_dag, task_id=test, execution_date=20220315T114500, start_date=20220315T164821, end_date=20220315T164821
[2022-03-15 11:48:21,597] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-03-15 11:48:21,625] {local_task_job.py:264} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2022-03-15 11:49:57,567] {taskinstance.py:1037} INFO - Dependencies all met for <TaskInstance: simple_dag.test scheduled__2022-03-15T11:45:00+00:00 [queued]>
[2022-03-15 11:49:57,575] {taskinstance.py:1037} INFO - Dependencies all met for <TaskInstance: simple_dag.test scheduled__2022-03-15T11:45:00+00:00 [queued]>
[2022-03-15 11:49:57,575] {taskinstance.py:1243} INFO - 
--------------------------------------------------------------------------------
[2022-03-15 11:49:57,576] {taskinstance.py:1244} INFO - Starting attempt 1 of 1
[2022-03-15 11:49:57,576] {taskinstance.py:1245} INFO - 
--------------------------------------------------------------------------------
[2022-03-15 11:49:57,585] {taskinstance.py:1264} INFO - Executing <Task(BashOperator): test> on 2022-03-15 11:45:00+00:00
[2022-03-15 11:49:57,589] {standard_task_runner.py:52} INFO - Started process 21582 to run task
[2022-03-15 11:49:57,596] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'simple_dag', 'test', 'scheduled__2022-03-15T11:45:00+00:00', '--job-id', '293', '--raw', '--subdir', 'DAGS_FOLDER/simple_dag.py', '--cfg-path', '/var/folders/tl/ljh9dpdd2fnbgmc62qc2b49h0000gn/T/tmpjkt3zg6g', '--error-file', '/var/folders/tl/ljh9dpdd2fnbgmc62qc2b49h0000gn/T/tmpn8qywe3j']
[2022-03-15 11:49:57,600] {standard_task_runner.py:77} INFO - Job 293: Subtask test
[2022-03-15 11:49:57,658] {logging_mixin.py:109} INFO - Running <TaskInstance: simple_dag.test scheduled__2022-03-15T11:45:00+00:00 [running]> on host 1.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.ip6.arpa
[2022-03-15 11:49:57,706] {taskinstance.py:1429} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=simple_dag
AIRFLOW_CTX_TASK_ID=test
AIRFLOW_CTX_EXECUTION_DATE=2022-03-15T11:45:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-03-15T11:45:00+00:00
[2022-03-15 11:49:57,708] {subprocess.py:62} INFO - Tmp dir root location: 
 /var/folders/tl/ljh9dpdd2fnbgmc62qc2b49h0000gn/T
[2022-03-15 11:49:57,710] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'echo "The DAG has ran!"']
[2022-03-15 11:49:57,723] {subprocess.py:85} INFO - Output:
[2022-03-15 11:49:57,729] {subprocess.py:89} INFO - The DAG has ran!
[2022-03-15 11:49:57,730] {subprocess.py:93} INFO - Command exited with return code 0
[2022-03-15 11:49:57,756] {taskinstance.py:1272} INFO - Marking task as SUCCESS. dag_id=simple_dag, task_id=test, execution_date=20220315T114500, start_date=20220315T164957, end_date=20220315T164957
[2022-03-15 11:49:57,815] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-03-15 11:49:57,841] {local_task_job.py:264} INFO - 1 downstream tasks scheduled from follow-on schedule check
